packages/neo3-parser/README.MD

```diff
@@ -10,6 +10,107 @@
 </p>
 
 # Neo3-parser
+
 Neo3-parser is a specification of how SmartContract client SDKs can interact with different parsing libraries such as Neon-JS.
 
 Visit the [main page](../../README.md) of the project.
+
+## How to parse responses
+
+After invoking a contract you'll get the results on a stack. Use the `parseRpcResponse(field: RpcResponse, parseConfig?: ParseConfig)` function on each item to get the results parsed.
```

Comment:
What do you mean by "on each item"? It sounds like if I have multiple variables on my response I will need to do this for each of them, and this is not true, I can do this once for my response and it will parse everything.

Reply:
Oh, I meant each item inside the result stack, e.g., if I invoked 3 methods on the same transaction, then the result stack would have length 3 and I would need to call parseRpcResponse 3 times. 

I could overload the function and have a `function parseRpcResponse(field: RpcResponse, parseConfig?: ParseConfig): any` and a `function parseRpcResponse(field: RpcResponse[], parseConfig?: ParseConfig[]): any[]` if you prefer

Reply:
In the majority of the use cases the user will invoke a single method on the transaction, so I think this overload is not necessary. I am just concern with the understanding. IDK, maybe saying "each item of the stack" might be enought to clarify.